<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Set cub</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P1xCub">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P1yCub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P2xCub">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P2yCub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P3xCub">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P3yCub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P4xCub">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P4yCub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P5yCub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P5xCub">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P6yCub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P6xCub">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P7yCub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P7xCub">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P7</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P8yCub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P8xCub">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buildCub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>157</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить куб</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P1zCub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P2zCub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P3zCub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P4zCub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P5zCub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P6zCub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P7zCub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P8zCub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P00xCub">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P11xCub">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P00zCub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P10xCub">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P(1,1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P(0,0)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P01xCub">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P(1,0)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P11yCub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P(0,1)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P01yCub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P10yCub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P00yCub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P10zCub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P01zCub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P11zCub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>35</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="displayBoundaries">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>157</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Отобразить границы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buildSurface">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>157</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить поверхность</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
